nginx.conf

```diff
@@ -50,25 +52,38 @@ http {
             return 200 '{"status":"UP"}';
         }
     }
-
     server {
         listen 8081;
         root /usr/share/nginx;
         location / {
             try_files /export/$uri /templated/export/$uri /templated/export/$uri/index.html =404;
         }
     }
-
     server {
         listen 8082;
         root /usr/share/nginx/recovery;
     }
-
     server {
         listen 8083;
         root /usr/share/nginx;
         location / {
             try_files /import/$uri /templated/import/$uri /templated/import/$uri/index.html =404;
         }
     }
+
+    # oauth
+    server {
+        listen 8084;
+        root /usr/share/nginx;
+        location / {
+            try_files /oauth-origin/$uri /templated/oauth-origin/$uri /templated/oauth-origin/$uri/index.html =404;
```

Comment:
I don't think this is needed because we don't have the need to template anything for these static pages!

(remove `/templated/oauth-origin/$uri /templated/oauth-origin/$uri/index.html`)